reference/constraints/Time.rst

```diff
@@ -1,9 +1,9 @@
 Time
 ====
 
-Validates that a value is a valid time, meaning either a ``DateTime`` object
-or a string (or an object that can be cast into a string) that follows
-a valid "HH:MM:SS" format.
+Validates that a value is a valid time, meaning either a ``DateTime`` object or
+an object implementing ``DateTimeInterface`` or a string (or an object that can
```

Comment:
This could be shortened a bit as the `DateTime` class implements `DateTimeInterface` too:

> [...] meaning an object implementing `DateTimeInterface` or a string [...]